### DISCUSSION TITLE
Very strange Greek characters problem

### DISCUSSION THREAD

**Discussion by mikekgr** [rustdesk/rustdesk#931] [General]
Dear Sirs,
I realized that there is a very strange problem. I use both English & Greek keyboard on my Windows 10 Enterprise laptop where RustDesk 1.1.9 is installed. Well, whatever I am connected to a remote RustDesk computer regardless if the remote computer is using Windows 10 or Linux (I have tested both) I cant put tone on a vowel ex in Greek language we can put a tone in Greek Alpha letter wich is: α   and if we put tone on it ά   to put tone of vowel we combine two keystrokes first we press ; key and then the vowel ex α  so ; and then α   give as  ά 
Please see it it is very important in Greek language.
Thanks !
P.S. While connected, I CAN NOT EVEN put tone on a vowel e.x. ά  In my **local Notepad**. If I just close the remote RustDesk connection the the normal functionality is returning

**Comment by chiehw** [comment]
Thanks for your feedback, I reproduced the problem.

**Reply by mikekgr** [reply]
Thanks for fast reaction. When we will have the next RustDesk release with this (and many other) problems solved?

**Reply by chiehw** [reply]
I already have a solution to this problem, which will be fixed in a while.

**Comment by mikekgr** [comment]
Dear RustDesk team, Please make a new version (even as beta) with this problem solved. It's very important for all Greek people !
Many thanks in advance

**Reply by rustdesk** [reply]
Allow us more time, we are doing a big refactory to solve international keyboard issues.

**Reply by mikekgr** [reply]
> Allow us more time, we are doing a big refactory to solve international keyboard issues.

Sure. Thanks a lot for your efforts to solve this issue.

**Reply by mikekgr** [reply]
Dear Sirs, yesterday I tested the current master of the git and I can say that the problem with the Greek characters still exist. As I wrote in my first post, I can't write Greek special alphabets as έ ύ ί ό ά ή ώ  when I type the keyboard combination in order to get these characters in the target screen I get ;έ ;ύ ;ί ;ό ;ά ;ή ;ώ that meas the RustDesk is sending the pressending key (that is the key ";" following with the correct tone of vowel e.x. "ά" bus not all the times, some tinmes the letter is simple e.x. "α" without tone..
Please try to fix it, it iis very importand for all Greek people. I am here to test things if you need so.
**Question: Sould I open a new issue in the git or the reported problem here is ok ?**
Best Regards,
Mike Kranidis

**Reply by rustdesk** [reply]
We are still working on the refactory job to solve such problem thoroughly, https://github.com/rustdesk/rustdesk/pull/964

**Reply by mikekgr** [reply]
@rustdesk 
Thanks for let me know. Do you need to open an issue on hte git or it is ok  the reference here?

**Reply by rustdesk** [reply]
too many such issues, we can not follow every one.

**Reply by rustdesk** [reply]
Please submit new issue if you see it again with our new release (comming soon or later).

**Reply by mikekgr** [reply]
@rustdesk 
Sure. What is the expecting release date?

**Reply by rustdesk** [reply]
I wanna know also.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/964
